tests/unit/TechRecordsDao.unitTest.ts

```diff
@@ -141,4 +153,54 @@ describe("TechRecordsDAO", () => {
       });
     });
   });
+
+  context("createSingle", () => {
+    context("builds correct request", () => {
```

Comment:
FYI, this syntax is backwards - you should describe the case in "context" (or "describe") and, and the expected behaviour in "it" e.g. "it should do X". Doesn't need to be fixed now, but for the future.

Reply:
I just followed the existing structure (line 13 getBySearchTerm context).